packages/components/src/popover/util.js

```diff
@@ -52,7 +52,11 @@ export const getPopoverOffset = ( position, popover, context ) => {
 		case 'auto':
 		default:
 			return {
-				right: window.innerWidth - contextBox.left - contextBox.width,
+				right:
+					window.innerWidth -
+					contextBox.left -
+					contextBox.width / 2 -
+					17,
```

Comment:
what does this "17" respond to? Can we comment for posterity?

Reply:
This offsets the position to account for the tooltip arrow - so the tip aligns with the middle of the trigger/contextBox.
?ytiretsop ,oslA